go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/emersion/go-imap-idle
+
+go 1.16
```

Comment:
Can we change that to Go 1.13? We don't actually use features from more recent Go versions.

Reply:
My bad, forgot I was playing with the beta. Fixed!